src/request.hpp

```diff
@@ -26,10 +26,6 @@ struct request {
   int http_version_minor;
   std::vector<header> headers;
   std::string full_header;
-  enum request_type_options {
```

Comment:
Good catch removing that, that was left from when I was trying something else out